src/neo-vm/OpCode.cs

```diff
@@ -3,398 +3,101 @@ namespace Neo.VM
     public enum OpCode : byte
     {
         // Constants
+
+        PUSHINT8 = 0x00,
+        PUSHINT16 = 0x01,
+        PUSHINT32 = 0x02,
+        PUSHINT64 = 0x03,
+        PUSHINT128 = 0x04,
+        PUSHINT256 = 0x05,
```

Comment:
From `0x06` to `0x0A` it will be reserved for the future?

Reply:
Yes.